capirca/lib/nsxt.py

```diff
@@ -137,69 +189,55 @@ def __str__(self):
 
     notes = ''
     if self.term.comment:
-      notes = self.term.comment
+      notes = '\n'.join(self.term.comment)
 
     action = 'ALLOW'
     if self.term.action:
       action = _ACTION_TABLE.get(self.term.action[0])
 
-    source_address = 'ANY'
+    source_address = ['ANY']
     if self.term.source_address:
       source_address = []
       for i in self.term.source_address:
         source_address.append(str(i))
 
-    destination_address = 'ANY'
+    destination_address = ['ANY']
     if self.term.destination_address:
       destination_address = []
       for i in self.term.destination_address:
         destination_address.append(str(i))
 
-    # Intentionally left with default values as these fields are mandatory in
-    # the API. If they need to be configured these should be extended to be
-    # read from the policy files.
-
-    services = 'ANY'
-
-    profiles = 'ANY'
-
-    scope = 'ANY'
-
-    if not self.term.destination_address:
-      direction = "IN"
-    elif not self.term.source_address:
-      direction = 'OUT'
-    else:
-      direction = 'IN_OUT'
-
-    tag = ''
-
-    ip_protocol = ''
-    if self.term.protocol:
-      ip_protocol = self.term.protocol
-
     rule = {
       "action": action,
       "resource_type": "Rule",
       "display_name": name,
       "source_groups": source_address,
       "destination_groups": destination_address,
-      "services": [
-          services
-      ],
-      "profiles": [
-          profiles
-      ],
-      "scope": [
-          scope
-      ],
+      "services": ["ANY"],
```

Comment:
Please document that `"services": ["ANY"],` means that `service_entries` gets used. (I'm referencing a comment from another contribution which doesn't have a PR open: https://github.com/google/capirca/compare/master...jcatrouillet:capirca:master#diff-c11ad0bd48e47a1421118f82ccb0b4bdc5b943a36389e9c0f745e3f3a850d879R169-R170)